README.md

```diff
@@ -18,3 +21,49 @@
 
 ### [Week 9](weeks/week09/week09.md): Advanced AI-Driven Automation Techniques
 
+### [Week 10](weeks/week10/week10.md): Building a Test Automation Suite, utilizing both web and API testing
+
+
+## Fork the Repo
+
+A `fork` is a new repository that shares code and visibility settings with the original “upstream” repository. This is a good way to have your own repository and still having updates from the "upstream". Here the basic steps to take it. Follow the references to you want to get it deeper.
+
+
+### Creating your fork
+
+1. Go to the repository to fork: https://github.com/ericrommel/python_test_automation_book
+2. Fork the repository (aka make a copy)
+3. From the forked repository, add the mentors as collaborators
+4. Clone the fork (aka download the copy)
+5. Make a git branch
+6. Make your changes
+7. Make your commits (aka do the work)
+8. Open/Create a Pull Request pointing to your forked `main` branch (aka PR)
+9. Set the mentors as reviewers
+10. Expect comments to fix or approvals
+11. Once your work is approved, you can merge in
+
+### Updating your fork
+
+The `main` code can change during the training. To get those changes into your fork, grab the `HTTPS` clone URL from the original repository (the home repository or upstream, not your fork) and do these commands:
+
+```bash
+$ git checkout main
+$ git remote add upstream the-clone-https-url
+$ git pull upstream main
+$ git push origin main
+```
+
+Commands explained:
+ - `git checkout main`: It is to switch to the forked `main` branch.
+ - `git remote add upstream the-clone-https-url`: It helps you to create a label called `upstream` that is connected to the URL for the `main` project. This is a one-time configuration. Once done, you don't need to it again next time.
```

Comment:
Once done, you don't need to **do** it again next time.

Reply:
Fixed!